pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -48,11 +48,11 @@ let
       nativeBuildInputs = [ bash coreutils which zip unzip ];
       patchPhase = ''
         substituteInPlace ortograf/herramientas/make_dict.sh \
-           --replace /bin/bash bash \
+           --replace /bin/bash /bin/sh \
```

Comment:
Is this used at build time or runtime? We should probably replace it with an absolute path to bash or runtimeShell